### PR TITLE
V3.1.0-2.46.2

### PR DESCRIPTION
**Pull request by sschneid**
* Upgrades Jenkins to 2.46.2 LTS
 * Updates various plugins to their latest versions

### CODE REVIEW

**Comment by sschneid** [comment]
@mesosphere-ci retest this please

**Comment by sschneid** [comment]
@mesosphere-ci retest this please

**Comment by sschneid** [comment]
Not merge-ready yet:
````
There are dependency errors loading some plugins:
GitHub Pipeline for Blue Ocean v1.0.0
Pipeline REST API for Blue Ocean v1.0.0 failed to load. Fix this plugin first.
Git Pipeline for Blue Ocean v1.0.0
Pipeline REST API for Blue Ocean v1.0.0 failed to load. Fix this plugin first.
Events API for Blue Ocean v1.0.0
Pipeline REST API for Blue Ocean v1.0.0 failed to load. Fix this plugin first.
Blue Ocean Pipeline Editor v0.2.0
Pipeline REST API for Blue Ocean v1.0.0 failed to load. Fix this plugin first.
Personalization for Blue Ocean v1.0.0
Events API for Blue Ocean v1.0.0 failed to load. Fix this plugin first.
Blue Ocean v1.0.0
Git Pipeline for Blue Ocean v1.0.0 failed to load. Fix this plugin first.
Pipeline REST API for Blue Ocean v1.0.0
Pipeline: Nodes and Processes v2.8 is older than required. To fix, install v2.10 or later.
Dashboard for Blue Ocean v1.0.0
Events API for Blue Ocean v1.0.0 failed to load. Fix this plugin first.
````

**Comment by sschneid** [comment]
I've tested this manually (`mesosphere/jenkins-dev:v3.1.0-2.46.2`) and things look 👍  to me.

File: scripts/test.sh [hunk 1]
```diff
@@ -54,9 +54,10 @@ CLUSTER_INFO=$(http                                         \
 )
 
 DCOS_URL="http://$(echo "${CLUSTER_INFO}" | jq -r ".DnsAddress")"
+echo "DCOS_URL is '${DCOS_URL}'"
 
-ln -s $DOT_SHAKEDOWN ~/.shakedown
-TERM=velocity shakedown --stdout all --ssh-key-file $CLI_TEST_SSH_KEY --dcos-url $DCOS_URL
+ln -s "${DOT_SHAKEDOWN}" ~/.shakedown
+TERM=velocity shakedown --stdout all --ssh-key-file "${CLI_TEST_SSH_KEY}" --dcos-url ${DCOS_URL}
```
Comment (colin-msphere) [inline scripts/test.sh]: Can you add quotes around `${DCOS_URL}` as well?

**Pull request merged**